src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/RazorHoverProvider.ts

```diff
@@ -37,10 +37,15 @@ export class RazorHoverProvider
             return;
         }
 
+        if (!applicableHover.range) {
+            // The range should always be defined, this is an error
+            return;
+        }
+
         // Re-map the projected hover range to the host document range
         const remappedResponse = await this.serviceClient.mapToDocumentRanges(
             projection.languageKind,
-            [applicableHover.range!],
+            [applicableHover.range],
```

Comment:
What would this entail?

Reply:
According to the doc-comment for that property: `When missing, the editor will use the range at the current position or the current position itself.`. Basically I'm just trying to trade out for a "safer" null-forgiveness here